src/pages/login/LoginPage.jsx

```diff
@@ -0,0 +1,11 @@
+import LoginCard from "../../components/login_signup/LoginCard"
+import Nav from "../../components/Nav/Nav"
```

Comment:
You can use an ```index.js``` file and add all your exports here and you can simply import all your component from a single file.

Reply:
Thanks for this suggestion, did this mistake in quite many files